.gitignore

```diff
@@ -4,3 +4,5 @@ examples/build.js
 gh-pages
 node_modules/
 npm-debug.log
+cert.crt
```

Comment:
nit: I don't think you should be using local cert and adding them to ignore, but instead using a server that could run the https by itself as webpack or so. Not sure if budo has that option though

Reply:
FYI, budo can generates a self-signed certificate if no `-cert` / `--key` are passed

Reply:
that's great to know thanks!

Reply:
Yep. Just discovered the project was running an old version of budo that required a cert and key. Updated that dep.